restapi.go

```diff
@@ -851,10 +870,38 @@ func (s *Session) GuildMemberEdit(guildID, userID string, roles []string) (err e
 // NOTE : I am not entirely set on the name of this function and it may change
 // prior to the final 1.0.0 release of Discordgo
 func (s *Session) GuildMemberMove(guildID, userID, channelID string) (err error) {
+	ns := NullString(channelID)
+	data := GuildMemberEditData{ChannelID: &ns}
 
-	data := struct {
-		ChannelID string `json:"channel_id"`
-	}{channelID}
+	_, err = s.RequestWithBucketID("PATCH", EndpointGuildMember(guildID, userID), data, EndpointGuildMember(guildID, ""))
+	if err != nil {
+		return
+	}
```

Comment:
This error check is redundant. I know there are some of these in the REST code currently, but it'd be better to not add any more. :)